WooCommerce/Classes/ViewModels/Order Details/Refunded Products/RefundedProduct.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+
+/// This model represents the aggregate data for refunded products.
+///
+final class RefundedProduct {
```

Comment:
I'm a bit confused, this model looks like exactly the same as `AggregateOrderItem`, could these two use cases share the same model if the difference is only how the properties populated?

Reply:
Deleting